LICENSE

```diff
@@ -276,6 +276,7 @@
           Copyright (c) 2015 by Contributors
           Copyright 1984, 1987, 1992 by Stephen L. Moshier
 
+    27. CNN Text Classification Example - For details, see example/cnn_text_classification/data_helpers.py
```

Comment:
The LICENSE file is more specifically our source release license. It should only refer to things which are bundled as part of the source release (http://www.apache.org/dev/licensing-howto.html). Maybe we could move this to a separate DATASET_LICENSE file?

Reply:
I'm changing the dataset license link as a comment of the code that downloads it. As these dataset are not bit included in our distribution, and should not be included in top level LICENSE file.